nibabel/gifti/giftiio.py

```diff
@@ -10,12 +10,15 @@
 # Stephan Gerhard, Oktober 2010
 ##############
 
-import os
-import codecs
+import warnings
 
-from .parse_gifti_fast import parse_gifti_file
+import numpy as np
 
 
+warnings.warn('Please use nibabel.load/save.', DeprecationWarning)
```

Comment:
`giftiio.py` is imported by `nibabel/gifti/__init__.py`. I would say either raise a `DeprecationWarning` when either `read` or `write` is called, or stop importing into `nibabel.gifti`. (I think you have to keep importing for API reasons, though?)